.github/workflows/test.yml

```
@@ -44,12 +44,12 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4
 
-      - name: Install NodeJS
+      - name: Set up NodeJS
         uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4
         with:
           node-version: "20"
 
-      - name: Install Prettier
+      - name: Set up Prettier
```

this is a install step
```suggestion
      - name: Install Prettier
```